modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/render.tsx

```diff
@@ -93,6 +93,9 @@ export default function render(
 		);
 	}
 	else {
+
+		// @ts-ignore
```

Comment:
What is this ignore for? 

Can you go through all ts-ignores you added and just make sure they are needed?